### ISSUE TITLE
Looking for clarification on entropy loss calculations for vtrace agent

### ISSUE THREAD

**Issue by Edvard-D** [google-research/seed_rl#73]
There are two entropy loss values that get calculated:

    entropy = tf.reduce_mean(
        parametric_action_distribution.entropy(learner_outputs.policy_logits))
    entropy_loss = tf.stop_gradient(agent.entropy_cost()) * -entropy

and

    if FLAGS.target_entropy:
        entropy_adjustment_loss = agent.entropy_cost() * tf.stop_gradient(
            tf.reduce_mean(entropy) - FLAGS.target_entropy)
    else:
        entropy_adjustment_loss = 0. * agent.entropy_cost()  # to avoid None in grad

I'm trying to understand why `tf.stop_gradient` isn't used consistently in these two calculations? For `entropy_loss` it's wrapped around the `agent.entropy_cost` call, but it's not wrapped around that same call when calculating `entropy_adjustment_loss`. It's wrapped around the `tf.reduce_mean` and subtraction when calculating `entropy_adjustment_loss`, but not wrapped around the `tf.reduce_mean` call when calculating `entropy_loss`.

I've been running into some problems with the entropy value exploding (sabotaging the training run), and I'm wondering if it's possible something isn't set up correctly here that might be contributing to it?
